tests/test.py

```diff
@@ -128,6 +128,14 @@ def fetch_json(self, path, params={}, **kwargs):
 # TODO: split the handler tests into another file
 
 
+class TestRootHandler(BaseTestCase):
+    def test_home_page(self):
+        response = self.fetch('/')
+        contents = response.body.decode('utf-8')
+        content_head = contents[contents.find('<h1>') + 4: contents.find('</h1>')]
```

Comment:
This is really specific. I would just check `'Apertium APy' in contents`.

Reply:
Okay I have changed that